powerauth-webflow-resources/src/main/java/io/getlime/security/powerauth/lib/webflow/resource/controller/UserProfileController.java

```diff
@@ -76,13 +77,15 @@ public UserProfileController(DataAdapterClient client, AuthorizationServerTokenS
             userResponse.getUser().setId(userDetail.getId());
             userResponse.getUser().setGivenName(userDetail.getGivenName());
             userResponse.getUser().setFamilyName(userDetail.getFamilyName());
+            userResponse.getConnection().setOrganizationId((String) additionalInfo.get(ORGANIZATION_ID));
```

Comment:
... this comment is more relevant in `WebFlowTokenEnhancer` class.

Reply:
Good point, I will check it.

Reply:
Looking at the code, I do not think there will be a problem, because the new attribute is stored in a HashMap, so we do not modify class structure, we just add another item into the HashMap. I will test it to make sure.

See:
https://github.com/wultra/powerauth-webflow/blob/10ec12098ad6a046ccd2fb0f306b1070b41623ed/powerauth-webflow/src/main/java/io/getlime/security/powerauth/app/webflow/oauth/WebFlowTokenEnhancer.java#L53

https://github.com/spring-projects/spring-security-oauth/blob/558a428d4637e0df2bab707b24e9e38620153220/spring-security-oauth2/src/main/java/org/springframework/security/oauth2/common/DefaultOAuth2AccessToken.java#L224